### DISCUSSION TITLE
Add use cases / providers to documentation

### DISCUSSION THREAD

**Discussion by WilliamDEdwards** [borgbackup/borg#8588] [Ideas]
While reading through the Borg documentation, I noticed that while Borg's technical advantages are clear, there is little mention of 'real-world usage'.

To make sure no one drops out, I think the documentation could be improved in two areas:

- List/recommend providers that offer Borg in a plug-and-play fashion. ([BorgBase](https://www.borgbase.com/) comes to mind; not affiliated with them, nor have any experience with them). This helps those interested in Borg's functionality, but not wanting to manage the Borg server.
- Mention common use cases. I can imagine people feel more confident picking Borg, knowing that Borg has stood the test in their specific case. For example, our hosting infrastructure branch Cyberfusion Core allows users to create backups with Borg (https://core-api.cyberfusion.io/redoc#tag/Borg-Repositories; backed by https://github.com/CyberfusionIO/python3-cyberfusion-borg-support). Skimming through the issues in this repository, there are many other use cases in which Borg has been proven, like homelabs and big data.

**Comment by ThomasWaldmann** [comment]
Hmm, this is kind of a scope issue. The scope of the borg docs is to document borg backup, not so much what happens outside / related to borg, that is rather the scope of the `borgbackup/community` repo.
